homeassistant/components/wallbox/lock.py

```diff
@@ -0,0 +1,86 @@
+"""Home Assistant component for accessing the Wallbox Portal API. The lock component creates a lock entity."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.components.lock import LockEntity, LockEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import InvalidAuth, WallboxCoordinator, WallboxEntity
+from .const import (
+    CONF_DATA_KEY,
+    CONF_LOCKED_UNLOCKED_KEY,
+    CONF_SERIAL_NUMBER_KEY,
+    DOMAIN,
+)
+
+
+@dataclass
+class WallboxLockEntityDescription(LockEntityDescription):
+    """Describes Wallbox sensor entity."""
```

Comment:
If you don't need additional keys, just use `LockEntityDescription` directly.

Reply:
OK, I changed that